src/main/webapp/js/instructorStudentList.js

```diff
@@ -24,13 +24,49 @@ $(document).ready(function() {
 
     // Binding for 'Show Emails' check box.
     $('#show_email').on('change', function() {
+        var $copyEmailButton = $('#copy-email-button');
+        var $emails = $('#emails');
+        
         if (this.checked) {
-            $('#emails').show();
+            $emails.show();
+            $copyEmailButton.show();
         } else {
-            $('#emails').hide();
+            $emails.hide();
+            $copyEmailButton.hide();
         }
+        
         filterEmails();
     });
+    
+    // Binding for copy email button
+    var copyEmailPopoverTimeout;
+    $('#copy-email-button').click(function(e) {
+        e.preventDefault();
+        clearTimeout(copyEmailPopoverTimeout);
+        var $copyEmailButton = $('#copy-email-button');
```

Comment:
Should be ok to do `var $this = $(this)` and replace the occurrences of `$copyEmailButton` with `$this`